docs/02.API-REFERENCE.md

```diff
@@ -12006,3 +12006,72 @@ main (void)
   return 0;
 }
 ```
+
+# Container Functions
+
+## jerry_get_buffer_from_container
```

Comment:
The real API method does not match the documentation "jerry_get_array_from_container " vs "jerry_get_buffer_from_container". The docs should be updated.